tests/acceptance/test_author_create_typehead.py

```diff
@@ -0,0 +1,62 @@
+
+# -*- coding: utf-8 -*-# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+
+from __future__ import absolute_import, division, print_function
+from time import sleep
+
+from selenium.webdriver.common.keys import Keys
+
+import os
```

Comment:
Imports should be:
```python
from __future__ import absolute_import, division, print_function

import os
from time import sleep

from selenium.webdriver.common.keys import Keys
```
the idea: first `__future__`, then stdlib, then vendored dependencies, then Invenio dependencies (not present here), then INSPIRE dependencies (not present here).